lib/tasks/factory_girl.rake

```diff
@@ -1,15 +0,0 @@
-namespace :factory_girl do
-  desc "Verify that all FactoryGirl factories are valid"
-  task lint: :environment do
-    if Rails.env.test?
-      begin
-        DatabaseCleaner.start
-        FactoryGirl.lint
-      ensure
-        DatabaseCleaner.clean
-      end
-    else
-      system("bundle exec rake factory_girl:lint RAILS_ENV='test'")
-    end
-  end
-end
```

Comment:
I see now that the project is not using Factory Girl. Do you think it would be good to keep this in the event that we eventually do use it?